verbit/streaming_client.py

```diff
@@ -250,6 +255,13 @@ def _connect_and_start(self,
             self._stop_media_thread = False
             self._media_sender_thread.start()
 
+        # start ping sender thread
+        if self.AUTO_PING_INTERVAL_SECONDS > 0:
+            self._ping_sender_thread = Thread(
+                target=self._ping_sender_worker,
+                name='ws_ping_sender')
+            self._ping_sender_thread.start()
```

Comment:
when we do reconnect after disconnecting for any reason
will we go though here again?

Reply:
yes, we get here after every reconnection. I checked it now to see that it's working well.
Since the `_ping_sender_worker` loop exits after the client is no longer connected the thread dies and when we get back here it just starts a new thread, and no zombie thread is left.

Reply:
After running unit-tests I realized that the thread worker implementation was not good enough and I decided to imitate the `WebSocketApp` auto ping thread worker implementation.
I tested it now multiple times and it seems to be working great.